src/components/List/List.jsx

```diff
@@ -1,22 +1,59 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import db from '../lib/firebase';
-import { collection, doc, updateDoc } from 'firebase/firestore';
+import db from '../../lib/firebase';
+import { collection, doc, updateDoc, query, orderBy } from 'firebase/firestore';
 import { useCollection } from 'react-firebase-hooks/firestore';
 import moment from 'moment';
-
-import { getEstimate, calcTimeDiff, formatDate } from '../helpers';
+import './list.css';
+import {
+  getEstimate,
+  calcTimeDiff,
+  formatDate,
+  calcDaysSince,
+} from '../../helpers';
 
 export const List = ({ token }) => {
   let navigate = useNavigate();
-  const [value, loading, error] = useCollection(collection(db, token), {
+  const q = query(collection(db, token), orderBy('item'));
+  const [value, loading, error] = useCollection(q, {
     snapshotListenOptions: { includeMetadataChanges: true },
   });
+
   let [filterText, setFilterText] = useState('');
+  let [items, setItems] = useState([]);
+
+  useEffect(() => {
+    if (value) {
+      let arr = [];
+      value.docs.forEach((item) => {
+        arr.push({ ...item.data(), id: item.id });
+      });
```

Comment:
This would be a great place to use `.map()`:

```suggestion
      let arr = value.docs.map((item) => ({ ...item.data(), id: item.id }));
```